xml/System/ReadOnlySpan`1.xml

```diff
@@ -424,9 +424,23 @@ This method returns `default` when `array` is `null`.
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>A read-only reference to an object of type T that can be used for pinning, and is required to support the use of span within a fixed statement.</summary>
+        <summary>Returns a read-only reference to an object of type T that can be used for pinning.
+        
+This method is intended to support .NET compilers and is not intended to be called by user code.</summary>
         <returns>A reference to the element of the span at index 0, or <see langword="null" /> if the span is empty.</returns>
-        <remarks>To be added.</remarks>
+        <remarks><format type="text/markdown"><![CDATA[
+
+## Remarks
+ Applications should not directly call `GetPinnableReference`. Instead, callers should use their language's normal pinning syntax, such as C\#'s `fixed` statement.
+
+ If pinning a `ReadOnlySpan<char>`, the resulting `char*` __is not__ assumed to be null-terminated. This behavior is different from pinning a `string`, where the resulting `char*` is guaranteed to be null-terminated.
+
+## Examples
+ The following example demonstrates creating an integer array, pinning it, and writing each element to the console.
+
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.span.getpinnablereference/cs/getpinnablereference1.cs" id="Snippet1":::
```

Comment:
```suggestion
:::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.span.getpinnablereference/cs/getpinnablereference1.cs":::
```

Since you're referencing the entire file, you don't need the snippet reference. You can also reference particular lines like this:

```
:::code language="csharp" source="Controllers/StudentsController.cs" range="2-24,26":::
```